src/kubeEvents.ts

```diff
@@ -0,0 +1,27 @@
+import { Kubectl } from './kubectl';
+import * as kubectlUtils from './kubectlUtils';
+import * as explorer from './explorer';
+
+export enum EventDisplayMode {
+    Show,
+    Follow
+}
+
+export async function getEvents(kubectl: Kubectl, displayMode: EventDisplayMode, explorerNode?: explorer.ResourceNode) {
+    let eventsNS;
+
+    if (explorerNode) {
+        eventsNS = explorerNode.id;
+    } else {
+        eventsNS = await kubectlUtils.currentNamespace(kubectl);
+    }
+
+    let cmd = `get events --namespace ${eventsNS}`;
+
+    if (EventDisplayMode.Follow === displayMode) {
```

Comment:
reverse the condition: `displayMode === EventDisplayMode.Follow`

Reply:
its a common practice to have non-nullable value in LHS, since this style is followed here I can reverse it